src/transport/SecureSessionMgr.cpp

```diff
@@ -92,136 +92,157 @@ CHIP_ERROR SecureSessionMgr::SendMessage(NodeId peerNodeId, System::PacketBuffer
 CHIP_ERROR SecureSessionMgr::SendMessage(PayloadHeader & payloadHeader, NodeId peerNodeId, System::PacketBufferHandle msgBuf,
                                          EncryptedPacketBufferHandle * bufferRetainSlot)
 {
-    return SendMessage(payloadHeader, peerNodeId, std::move(msgBuf), bufferRetainSlot, false);
+    PacketHeader packetHeader;
+    return SendMessage(payloadHeader, packetHeader, peerNodeId, std::move(msgBuf), bufferRetainSlot, false);
```

Comment:
what is the 'false' at the end?

Could we add a comment of /* isEncrypted */ at the end or maybe use some constants?